src/static/redirects.njk

```diff
@@ -31,6 +31,47 @@ eleventyExcludeFromCollections: true
+/docs/latest/user-guide/formatters/                          /docs/latest/use/formatters/ 301!
+/docs/latest/user-guide/integrations/                        /docs/latest/use/integrations/ 301!
+/docs/latest/user-guide/migrating-to-8.0.0/                  /docs/latest/use/migrate-to-8.0.0/ 301!
+/docs/latest/developer-guide/architecture/                   /docs/latest/contribute/architecture/ 301!
+/docs/latest/developer-guide/source-code/                    /docs/latest/contribute/development-environment/ 301!
+/docs/latest/developer-guide/development-environment/        /docs/latest/contribute/development-environment/ 301!
+/docs/latest/developer-guide/unit-tests/                     /docs/latest/contribute/tests/ 301!
+/docs/latest/developer-guide/working-with-rules/             /docs/latest/extend/custom-rules/ 301!
+/docs/latest/developer-guide/working-with-plugins/           /docs/latest/extend/plugins/ 301!
+/docs/latest/developer-guide/working-with-custom-formatters/ /docs/latest/extend/custom-formatters/ 301!
+/docs/latest/developer-guide/working-with-custom-parsers/    /docs/latest/extend/custom-parsers/ 301!
+/docs/latest/developer-guide/code-path-analsis/              /docs/latest/extend/code-path-analsis/ 301!
+/docs/latest/developer-guide/nodejs-api/                     /docs/latest/integrate/nodejs-api/ 301!
+/docs/latest/developer-guide/contributing/                   /docs/latest/contribute/ 301!
+/docs/latest/developer-guide/contributing/reporting-bugs/    /docs/latest/contribute/report-bugs/ 301!
+/docs/latest/developer-guide/contributing/new-rules/         /docs/latest/contribute/propose-new-rule/ 301!
+/docs/latest/developer-guide/contributing/rule-changes/      /docs/latest/contribute/propose-rule-change/ 301!
+/docs/latest/developer-guide/contributing/changes/           /docs/latest/contribute/request-change/ 301!
+/docs/latest/developer-guide/contributing/working-on-issues/ /docs/latest/contribute/work-on-issue/ 301!
+/docs/latest/developer-guide/contributing/pull-requests/     /docs/latest/contribute/pull-requests/ 301!
+/docs/latest/maintainer-guide/                               /docs/latest/maintain/ 301!
+/docs/latest/maintainer-guide/issues/                        /docs/latest/maintain/manage-issues/ 301!
+/docs/latest/maintainer-guide/pullrequests/                  /docs/latest/maintain/review-pull-requests/ 301!
+/docs/latest/maintainer-guide/releases/                      /docs/latest/maintain/manage-releases/ 301!
+/docs/latest/maintainer-guide/governance/                    /docs/latest/contribute/governance/ 301!
+/docs/latest/user-guide/configuring/*                        /docs/latest/use/contribute/:splat/ 301!
+/docs/latest/user-guide/*                                    /docs/latest/use/:splat/ 301!
+/docs/latest/developer-guide/*                               /docs/latest/extend/:splat/ 301!
+/docs/latest/maintainer-guide/*                              /docs/latest/maintain/:splat/ 301!
```

Comment:
When using `:splat`, leave off the trailing slash, otherwise we'll get two trailing slashes on the redirect in the browser URL bar.

```suggestion
/docs/latest/user-guide/configuring/*                        /docs/latest/use/contribute/:splat 301!
/docs/latest/user-guide/*                                    /docs/latest/use/:splat 301!
/docs/latest/developer-guide/*                               /docs/latest/extend/:splat 301!
/docs/latest/maintainer-guide/*                              /docs/latest/maintain/:splat 301!
```